[2023-05-02 18:40:49,277] INFO _log() L224  192.168.0.12 - - [02/May/2023 18:40:49] "GET /openapi/static/images/redoc.svg HTTP/1.1" 200 - - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 18:40:49,347] INFO _log() L224  192.168.0.12 - - [02/May/2023 18:40:49] "GET /openapi/static/images/apidoc.svg HTTP/1.1" 200 - - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 18:40:52,665] INFO _log() L224  192.168.0.12 - - [02/May/2023 18:40:52] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 18:40:52,718] INFO _log() L224  192.168.0.12 - - [02/May/2023 18:40:52] "GET /openapi/static/css/swagger-ui.css HTTP/1.1" 200 - - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 18:40:52,721] INFO _log() L224  192.168.0.12 - - [02/May/2023 18:40:52] "GET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1" 200 - - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 18:40:52,729] INFO _log() L224  192.168.0.12 - - [02/May/2023 18:40:52] "GET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1" 200 - - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 18:40:53,328] INFO _log() L224  192.168.0.12 - - [02/May/2023 18:40:53] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 18:40:53,346] INFO _log() L224  192.168.0.12 - - [02/May/2023 18:40:53] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 18:41:15,680] WARNING add_pesquisa() L62   Erro ao adicionar pesquisa 'Bruno de Souza', Não foi possível salvar novo item :/ - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\app.py L62  
[2023-05-02 18:41:15,682] INFO _log() L224  192.168.0.12 - - [02/May/2023 18:41:15] "[31m[1mPOST /pesquisa HTTP/1.1[0m" 400 - - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 18:41:19,365] WARNING add_pesquisa() L56   Erro ao adicionar pesquisa 'Bruno de Souza', Pesquisa de mesmo nome já salvo na base :/ - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\app.py L56  
[2023-05-02 18:41:19,367] INFO _log() L224  192.168.0.12 - - [02/May/2023 18:41:19] "[31m[1mPOST /pesquisa HTTP/1.1[0m" 409 - - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 18:43:04,311] WARNING add_pesquisa() L62   Erro ao adicionar pesquisa 'João Silva', Não foi possível salvar novo item :/ - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\app.py L62  
[2023-05-02 18:43:04,312] INFO _log() L224  192.168.0.12 - - [02/May/2023 18:43:04] "[31m[1mPOST /pesquisa HTTP/1.1[0m" 400 - - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 18:43:08,889] WARNING add_pesquisa() L56   Erro ao adicionar pesquisa 'João Silva', Pesquisa de mesmo nome já salvo na base :/ - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\app.py L56  
[2023-05-02 18:43:08,891] INFO _log() L224  192.168.0.12 - - [02/May/2023 18:43:08] "[31m[1mPOST /pesquisa HTTP/1.1[0m" 409 - - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 18:44:48,161] WARNING add_pesquisa() L62   Erro ao adicionar pesquisa 'Joao', Não foi possível salvar novo item :/ - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\app.py L62  
[2023-05-02 18:44:48,162] INFO _log() L224  192.168.0.12 - - [02/May/2023 18:44:48] "[31m[1mPOST /pesquisa HTTP/1.1[0m" 400 - - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 18:44:57,847] INFO _log() L224  192.168.0.12 - - [02/May/2023 18:44:57] "[31m[1mGET /pesquisa HTTP/1.1[0m" 405 - - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 18:45:50,469] ERROR log_exception() L1449 Exception on /pesquisa [DELETE] - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\flask\app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\flask_cors\extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\flask\app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\flask\app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\flask_openapi3\scaffold.py", line 355, in wrapper
    resp = _do_wrapper(
  File "C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\flask_openapi3\do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
  File "C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\app.py", line 98, in del_pesquisa
    pesquisa_nome = unquote(unquote(query.nome))
AttributeError: 'PesquisaBuscaSchema' object has no attribute 'nome'
[2023-05-02 18:45:50,494] INFO _log() L224  192.168.0.12 - - [02/May/2023 18:45:50] "[35m[1mDELETE /pesquisa?time=Clube%20de%20Regatas%20do%20Flamengo HTTP/1.1[0m" 500 - - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 18:45:52,945] ERROR log_exception() L1449 Exception on /pesquisa [DELETE] - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\flask\app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\flask_cors\extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\flask\app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\flask\app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\flask_openapi3\scaffold.py", line 355, in wrapper
    resp = _do_wrapper(
  File "C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\flask_openapi3\do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
  File "C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\app.py", line 98, in del_pesquisa
    pesquisa_nome = unquote(unquote(query.nome))
AttributeError: 'PesquisaBuscaSchema' object has no attribute 'nome'
[2023-05-02 18:45:52,951] INFO _log() L224  192.168.0.12 - - [02/May/2023 18:45:52] "[35m[1mDELETE /pesquisa?time=Clube%20de%20Regatas%20do%20Flamengo HTTP/1.1[0m" 500 - - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 18:54:29,709] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.0.12:5000 (Press CTRL+C to quit) - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 18:54:34,755] INFO _log() L224  127.0.0.1 - - [02/May/2023 18:54:34] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 18:54:34,837] INFO _log() L224  127.0.0.1 - - [02/May/2023 18:54:34] "GET /openapi/ HTTP/1.1" 200 - - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 18:54:34,979] INFO _log() L224  127.0.0.1 - - [02/May/2023 18:54:34] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 18:54:34,981] INFO _log() L224  127.0.0.1 - - [02/May/2023 18:54:34] "[36mGET /openapi/static/images/redoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 18:54:34,989] INFO _log() L224  127.0.0.1 - - [02/May/2023 18:54:34] "[36mGET /openapi/static/images/rapidoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\Bruno\Documents\PÓS-GRADUAÇÃO\backend\env\lib\site-packages\werkzeug\_internal.py L224 
